app/assets/javascripts/application/slides.js

```diff
@@ -1,12 +1,17 @@
 Utils.fireWhenReady(['slides#preview'], function(e) {
   // Scale the view to the size it will need to display
-  $('.ui-feed').css({width: '1920px', height: '1080px'});
+  var $uiFeed = $('.ui-feed');
+  var isVertical = $uiFeed.find(".js-slide-meta").data("orientation") == "vertical" ? true : false;
```

Comment:
This line could be:

   var isVertical = $uiFeed.find(".js-slide-meta").data("orientation") === "vertical";

Usually want to use strict equality with JS though I don't know that it matters here.